### PR TITLE
Mobile first portfolio pull request

### PR DESCRIPTION
**Pull request by AbdusSattar-70**
Added toolbar and headline section from development branch to master.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <!-- custom style link -->
+<link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+     <!-- header start -->
+  <header class="navbar-container">
+        <h1>John Doe</h1>
```
Comment (NduatiKagiri) [inline index.html]: - [x] The page title/logo should be a link and not a heading. Please make this change.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    </ul>
+
+  </div>
+
+
+  <div class="headline-image">
+
+    <img src="media/images/Illustration-header.png" alt="">
+
+  </div>
```
Comment (NduatiKagiri) [inline index.html]: - [x] You have done a great job on this headline section, apart from this image which you added as an image but should be added as a background image.

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @AbdusSattar-70 ,

# CHANGES REQUIRED ♻️   

- [ ] **[@##](jkkkkk)**

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights ✔️
✅ GitHub Flow observed
✅ Linters checks are passing
✅ Good documentation

## Required Changes ♻️

- [ ] You have not applied the sizes, padding, and margins from the Figma design.
-   Your design:
     johndoe1

-   Required Figma design:
    johndoe2

-   Kindly make sure that you remove the white spaces surrounding your design.
-   Make sure that your header occupies the whole width.
-    Please apply the right fonts for all the texts, and the right sizes for all the texts and images, as from the Figma design.
-    Kindly also remember to apply the right alignment of your content (that is center, right, left, bottom, end, top, and start), and also apply the right paddings and margins on all your elements. 


- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,2 +1,185 @@
...
+
+**[My-portfolio]** is my portfolio website that is made as a part of Microverse's task although I'll use it for my portfolio when will be completed. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
```
Comment (AbdusSattar-70) [inline README.md:42]: @AbdusSattar-70  problem is here.
repository-open-graph-template

File: src/html/index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            </li>
+            <li>
+              <a href="">
+                <img src="media/icons/twitter.svg" alt="twitter icon" />
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <img src="media/icons/medium.png" alt="medium icon" />
+              </a>
```
Comment (AbdusSattar-70) [inline src/html/index.html:80]: > `_****_`
```suggestion
                <img src="media/icons/twitter.svg" alt="twitter icon" />
              </a>
            </li>
            <li>
              <a href="">
                <img src="media/icons/medium.png" alt="medium icon" />
              </a> [optional] you should try another one.
```

File: src/html/nav-mobile.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+          <img src="/src/media/icons/cross.png" alt="" />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li><a href="index.html">Portfolio</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
```
Comment (AbdusSattar-70) [inline src/html/nav-mobile.html:27]: #problem

**Review by AbdusSattar-70 — COMMENTED** [review]
#4  please make sure

**Pull request closed without merging**